From 13d8c8c3aef01d194abf9c7c2c601e94b8d6732e Mon Sep 17 00:00:00 2001
From: Andrew Eikum <aeikum@codeweavers.com>
Date: Fri, 12 Nov 2021 14:21:08 -0600
Subject: [PATCH] wine.inf: Use built-in atiadlxx for Age of Empires IV.

CW-Bug-ID: #19602

Avoids the warning about too old drivers on start.
---
 loader/wine.inf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index c9371ee3882..c54af68914b 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -4238,6 +4238,7 @@ HKCU,Software\Wine\AppDefaults\Paradox Launcher.exe\DllOverrides,"libglesv2",,"d
 HKCU,Software\Wine\AppDefaults\Nickelodeon All-Star Brawl.exe\DllOverrides,"winusb",,"disabled"
 HKCU,Software\Wine\AppDefaults\nw.exe\DllOverrides,"libglesv2",,"disabled"
 HKCU,Software\Wine\AppDefaults\gotg.exe\DllOverrides,"atiadlxx",,"builtin"
+HKCU,Software\Wine\AppDefaults\RelicCardinal.exe\DllOverrides,"atiadlxx",,"builtin"
 HKCU,Software\Wine\AppDefaults\msedgewebview2.exe,"Version",,"win81"
 HKCU,Software\Wine\AppDefaults\ForzaHorizon5.exe\DllOverrides,"amd_ags_x64",0x2,"builtin"
 ;;Likely want *80 and *90 too, but those require removing Wine's manifest files.

